### PR TITLE
publish py.typed to the right place

### PR DESCRIPTION
**Pull request by dimbleby**
py.typed is supposed to live at the package root, currently users of this library are not seeing the types that it exposes.

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -1,3 +1,3 @@
 include CHANGELOG.md
-include py.typed
+include src/hijri_converter/py.typed
```
Comment (mhalshehri) [inline MANIFEST.in:2]: use `src/*/py.typed` instead

**Review by mhalshehri — COMMENTED** [review]
Please use the `develop` base branch for pull request

**Comment by dimbleby** [comment]
done

**Comment by dimbleby** [comment]
workflow seems to be hitting https://github.com/psf/black/issues/2964, looks like you'll want to update black

**Comment by mhalshehri** [comment]
Better to keep this pull request clean. I will ignore quality checks for now and mere you pull request.
Then I'll update `requirements-dev.txt` later.

**Comment by dimbleby** [comment]
... so shall I remove the commit that updates black?

**Comment by mhalshehri** [comment]
Thank you for your valuable contribution.

**Pull request merged**